src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let reset = false;
+  let counter = 0;
+
+  return function(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3) {
+        return a + b;
+      } else {
+        reset = !reset;
+        counter = 0;
+
+        if (reset) {
+          return 'Bzzz... Error!';
+        } else {
+          return a + b;
+        }
```

Comment:
```suggestion
      if (counter % 2 === 0 && counter > 3) {
        return 'Bzzz... Error!';
      }

      return a + b;
```

You can do it a little bit simpler: